package.json

```diff
@@ -1,18 +1,19 @@
 {
   "name": "@leadhome/oxygen",
-  "version": "0.1.2",
+  "version": "0.1.3",
   "description": "A react component library",
-  "main": "build/oxygen.cjs.js",
-  "module": "build/oxygen.esm.js",
+  "main": "build/oxygen.js",
   "scripts": {
-    "build": "rollup -c",
     "commit": "npx git-cz",
-    "start": "rollup -c -w",
     "storybook": "start-storybook -p 9001 -c .storybook",
     "deploy-storybook": "storybook-to-ghpages",
     "lint": "eslint -c .eslintrc src",
     "build-storybook": "build-storybook",
-    "semantic-release": "semantic-release"
+    "semantic-release": "semantic-release",
+    "start": "npm run storybook",
+    "prebuild": "npm run clean",
+    "build": "webpack --mode production --config webpack/config.js",
+    "clean": "rimraf build"
```

Comment:
What does this and prebuild do? I've never had to use either.

Reply:
`rimraf` is a npm package which allows you to delete a directory on both unix & windows. `rm -rf` only works on unix for example.

The prebuild hook just fires the clean script (which is the rimraf thing) to just clean the directory.
It's probably not completely necessary to keep deleting and recreating the folder, but it just keeps things a lot cleaner, especially when working with different outputs / tweaking the config.